itea.html

```diff
@@ -35,8 +35,8 @@ <h1>IS Tea Round Cheat Sheet</h1>
                             <tr><td><strong>Jen</strong></td><td> Always check, can change, occasionally has a coffee. Likes WHITE TEA (preference: weak, milky).</td></tr>
                             <tr><td><strong>Alex</strong></td><td> Does not drink tea or coffee.</td></tr>
                             <tr><td><strong>Suki</strong></td><td> Is a little teapot, short and stout.</td></tr>
-                            <tr><td><strong>Mila</strong></td><td> Morning is 2 x BLACK COFFEE with 2 x SUGAR. Sometimes STRONG BLACK TEA (two teabags, leave them in the cup, bring a spoon).</td></tr>
-                            <tr><td><strong>Jenny</strong></td><td> WHITE COFFEE.</td></tr>
+                            <tr><td><strong>Mila</strong>  <span class="label label-primary">New</span></td><td> Morning is 2 x BLACK COFFEE with 2 x SUGAR. Sometimes STRONG BLACK TEA (two teabags, leave them in the cup, bring a spoon).</td></tr>
+                            <tr><td><strong>Jenny</strong>  <span class="label label-primary">New</span></td><td> WHITE COFFEE.</td></tr>
```

Comment:
This will work while they're still new.
But what happens when they're not.
How can you untangle this potential Jenny-confusion?

Reply:
@gchristian17 very true. This is a stop-gap for now. Not sure how to untangle. What if we had more than one Gary in the team...

Raise an issue, we'll need to think of a longer-term fix.

Reply:
I once worked somewhere where there were four Garys. True story. In a team of 40-50.